bash_scripting_101/bash_scripting_101.md

```diff
@@ -274,4 +289,4 @@ We ask you to fill out a brief (5 minutes or less) survey to let us know:
 - If the module difficulty was appropriate
```

Comment:
Grab the Feedback structure from the most recent sample module, I think it is a little different (links yaml, etc)